app/javascript/packs/backup-code-analytics.js

```diff
@@ -0,0 +1,9 @@
+import { trackEvent } from '@18f/identity-analytics';
```

Comment:
Since we [default to TypeScript-first](https://github.com/18F/identity-idp/blob/main/docs/frontend.md#at-a-glance-1), can you make this a `.ts` file?